src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -960,7 +963,7 @@ else if ( CompilerConfiguration.CompilerReuseStrategy.ReuseSame.getStrategy().eq
                 }
 
                 String[] cl = compiler.createCommandLine( compilerConfiguration );
-                if ( getLog().isDebugEnabled() && cl != null && cl.length > 0 )
+                if ( cl != null && cl.length > 0 )
```

Comment:
I would prefer to keep the `getLog().isDebugEnabled()` here, otherwise a lot a String concatenation is done for showing the commandline, which is only shown during debug

Reply:
We've already performed this check here: https://github.com/apache/maven-compiler-plugin/pull/21/files#diff-d4e94a0220ac4630c8e2ca12df1b46bcR929

Reply:
Ah, good catch!